pegasus/test/test_hoc_routes.rb

```diff
@@ -105,6 +105,33 @@ class HocRoutesTest < Minitest::Test
       end
     end
 
+    it 'starts and ends given tutorial, tracking company and tutorial' do
```

Comment:
this test has the same name as the one above it. are we pretty sure both test are getting run?